new/LIT7115LanceWounds.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7115LanceWounds" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጸሎት፡ በእንተ፡ ኵናት፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Ṣalot baʾǝnta kʷǝnāt ...</title>
+                <title xml:lang="en" corresp="#t1">Prayer against the lance ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BLorient12859"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Magic prayer against lance wounds containing <foreign xml:lang="gez-tr">ʾasmāt</foreign> and
+                    <foreign xml:lang="gez-tr">qālāt</foreign>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Magic"/>
+                    <term key="Medicine"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="gez-tr">Gǝʿǝz transliterated</language>
```

Comment:
On rethinking - please do not use gez-tr. It was used for manuscripts/inscriptions using geez not in fidal, not for transcription we produce

Reply:
This makes sense and is good to know. Thank you!

Reply:
Would it be possible to add this to the "definition" of gez-tr that appears in Oxygen when this is suggested as a potential value of `<language ident="">` (see screenshot below)? I think it would be helpful for future encoders.

<img width="722" alt="Skärmavbild 2024-10-24 kl  08 37 10" src="https://github.com/user-attachments/assets/7d10f8bb-dfa7-40f6-9bd4-1ec46130639b">

Reply:
Maybe "Transliterated Gǝʿǝz when appearing in manuscripts or inscriptions (not for the transcription we produce)"